### ISSUE TITLE
`indentation` never measures the closing brace of a block whose last at-rule carries neither a block nor a semicolon

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#509]
### What minimal example or steps are needed to reproduce the bug?

Where the last node of a block is an at-rule carrying neither a block nor a semicolon of its own, that at-rule runs to the closing brace, so PostCSS files the whitespace in front of the brace into the at-rule's `raws.between` and leaves the block's `raws.after` empty — the shape #292 named for the three `block-closing-brace-*-before` rules, which read it through `getBlockAfter` since then.

`indentation` never learned it. The closing brace is measured off the block's `raws.after`, which holds no line break here, so the line the brace stands on is measured by nobody: the text of the at-rule the rule reads runs through `raws.between`, but is trimmed before the search, and the trim takes the whitespace closing the raw off its end — the brace's own line with it.

```css
a {
	@extend .b
		}
```

Under `"tab"` that draws no warning on any syntax, and `--fix` leaves it as it is. The same block with a declaration or a semicolon behind the at-rule reports `3:3` as `Expected indentation of 0 tabs` and is fixed on the first run:

| input | warnings |
| --- | --- |
| `a {\n\t@extend .b\n\t\t}` | none |
| `a {\n\tcolor: pink;\n\t\t}` | `3:3 Expected indentation of 0 tabs` |
| `a {\n\t@extend .b;\n\t\t}` | `3:3 Expected indentation of 0 tabs` |
| `@media x {\n\ta {\n\t\t@extend .b\n}\n}` | none |
| `a {\n\t@extend .b\n}` under `indentClosingBrace: true` | none |
| `a {\n\tcolor: pink;\n}` under `indentClosingBrace: true` | `3:1 Expected indentation of 1 tab` |

Measured on `0f1a6e5`, the branch of #375, under `postcss`, `postcss-scss` and `postcss-less` alike; a comment the at-rule swallowed in front of the brace changes nothing about the brace's line.

### What minimal configuration is needed to reproduce the bug?

```json
{
	"plugins": ["@stylistic/stylelint-plugin"],
	"rules": {
		"@stylistic/indentation": "tab"
	}
}
```

### What did you expect to happen?

The closing brace of such a block is measured and fixed like the closing brace of any other, wherever the parser filed the run in front of it: `getBlockAfter` and `setBlockAfter` already read and write that run for the three brace rules.

### What actually happened?

The brace's line goes unmeasured, so a mis-indented closing brace behind a body-less at-rule without a semicolon is neither reported nor fixed.

### Notes

Found while closing #375, which is the write into the same raw: the lines the at-rule swallowed in front of the brace are measured and now written in front of the line they belong to, and the brace's line is the one the trim leaves out. The two do not overlap — the writer of #375 writes at the head of a line inside `raws.between`, a fix for this one would write its tail through `setBlockAfter`.
